packages/dev/docs/pages/react-aria/testing.mdx

```diff
@@ -0,0 +1,231 @@
+{/* Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout, VersionBadge} from '@react-spectrum/docs';
+export default Layout;
+import testUtilDocs from 'docs:@react-aria/test-utils';
+import combobox from 'docs:@react-aria/test-utils/src/combobox.ts';
+import gridlist from 'docs:@react-aria/test-utils/src/gridlist.ts';
+import menu from 'docs:@react-aria/test-utils/src/menu.ts';
+import select from 'docs:@react-aria/test-utils/src/select.ts';
+import table from 'docs:@react-aria/test-utils/src/table.ts';
+import {ClassAPI, FunctionAPI, InterfaceType, TypeContext, TypeLink} from '@react-spectrum/docs';
+
+---
+category: Concepts
+---
+
+# Testing
+
+This page describes how to test an application built with React Aria. It documents the available testing utilities available
+for each aria pattern and how they can be used to simulate common user interactions.
+
+## Introduction
+
+Running automated tests on your application helps ensure that it continues to work as expected over time.
+You can use testing tools like [Enzyme](https://enzymejs.github.io/enzyme/) or
```

Comment:
We should probably remove enzyme at this point. I don't think it is maintained and doesn't work with newer versions of React.